tests/slow/BulkDumpingS3.toml

```diff
@@ -0,0 +1,78 @@
+[configuration]
+storageEngineExcludeTypes = [5] #FIXME: remove after allowing to do bulkloading with fetchKey and shardedrocksdb
+
+# Do not support tenant, encryption, and TSS
+tenantModes = ['disabled']
+encryptModes = ['disabled']
+disableTss = true # TODO(BulkLoad): support TSS. finishMoveShard should wait for TSS if a data move is bulkload data move.
+
+# Instruct simulation to generate a simple, single-region config
+generateFearless = false
+simpleConfig = false
+
+# Simple network configuration - single region, no satellites
+minimumRegions = 1
+# Use ssd-2 storage engine instead of rocksdb to avoid RocksDB bulk loading crashes
+# The RocksDB assertion failure indicates incompatibility with current bulk loading implementation
```

Comment:
Which assertion failure? The current bulkload implementation should be compatible to RocksDB. Please let me know more detail about this. Thanks!